test/e2e/Tests/Tests/ErrorHandlingTests.cs

```diff
@@ -21,6 +21,7 @@ public ErrorHandlingTests(FunctionAppFixture fixture, ITestOutputHelper testOutp
     }
 
     [Fact]
+    [Trait("MSSQL", "Skip")] // This test fails for MSSQL this bug is fixed: https://github.com/microsoft/durabletask-mssql/issues/287
```

Comment:
This test fails for MSSQL this bug is fixed
```suggestion
    [Trait("MSSQL", "Skip")] // This test fails for MSSQL unless this bug is fixed: https://github.com/microsoft/durabletask-mssql/issues/287
```